include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h

```diff
@@ -537,9 +537,12 @@ namespace internal
   class FineDoFHandlerView
   {
   public:
-    FineDoFHandlerView(const MeshType &mesh_fine, const MeshType &mesh_coarse)
+    FineDoFHandlerView(const MeshType &   mesh_fine,
+                       const MeshType &   mesh_coarse,
```

Comment:
Not directly related to this PR, but can you explain why you use the `MeshType` template argument? Given that we only have a single `DoFHandler` type, and that you explicitly use the `get_dof_indices` of the associated `cell_iterator`, I would prefer to spell the type and template parameters out.

Reply:
Simple explanation: this code backs back from a time when this was not the case^^ And yes, I have been planing to remove this, but was not able to do so because there were always some open PRs that did prevent in doing so. Once this PR is merged, I will remove the MeshTypes.